src/content/post/2025-04-15-embedded-dbs.md

```diff
@@ -13,20 +13,19 @@ tags: ["sqlite", "kuzu", "duckdb", "lancedb", "embedded dbms", "in-process dbms"
 So, in-process DBMSs are hardly new. Yet, starting with [DuckDB](https://github.com/duckdb/duckdb) and later by [Kuzu](https://github.com/kuzudb/kuzu) and [LanceDB](https://github.com/lancedb/lancedb), there is a 
 renewed interest in using in-process DBMSs within a wide range of data-intensive application domains: 
 relational analytics, graph querying and analytics, and vector search.
-As a co-founder of [Kuzu Inc.](https://kuzudb.com/) that develops and commercializes the Kuzu graph DBMS, 
-I get to answer several frequently asked questions about in-process DBMSs:
+As a co-founder of [Kuzu Inc.](https://kuzudb.com/), the team behind the Kuzu graph DBMS, 
+I get asked a lot of questions about this new generation of in-process databases, such as:
 
 - What are the advantages and disadvantages of in-process DBMSs?
 - What are their common usage scenarios? 
-- Does in-process DBMS mean an ephemeral DBMS ? (short answer: no but many offer an "in-memory" mode where databases are ephemeral)
-- Does in-process DBMS mean it can manage small data?  (no!) 
+- Does "in-process" mean the DBMS is ephemeral? (Short answer: no--but many offer an "in-memory" mode where databases are ephemeral)
```

Comment:
Does "in-process" mean the DBMS is ephemeral? -> Does "in-process" mean the databases are ephemeral?